nss-day-cohort-48/rare-client-polite-pirates
# view all posts

#### Changes Made
1. Added file `PostProvider.js` to `posts` directory & modified the file to include `getPosts` and `getPostsById` functions.
1. Added file `PostList.js` to `posts` directory & modified the file to include `getPosts` functionality.
1. Added file `PostDetail.js` to `posts` directory & modified the file to include `getPostsById` functionality.
1. Modified file `ApplicationViews.js` to include  url path`/posts`.
   ​
#### Steps to Review
1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout cjw-view-my-posts
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
- [ ] run python server in debugger panel on server side
- [ ] type in `npm start` for the client terminal
- [ ] click on `Posts` in the nav bar
- all posts should appear

4. View code file.
   > Confirm file modifications are present as indicated above.
